apps/web/package.json

```diff
@@ -89,7 +89,8 @@
     "less-loader": "4.1.0",
     "nodemon": "^2.0.15",
     "start-server-and-test": "1.11.6",
-    "webpack": "4.46.0"
+    "webpack": "4.46.0",
+    "react-error-overlay": "6.0.9"
```

Comment:
this fixes the occasional error react put in console.log of `process` is not defined.